tuna/miopen/driver/base.py

```diff
@@ -109,7 +114,7 @@ def construct_driver(self, line: str) -> bool:
     self.config_set_defaults()
     return True
 
-  def construct_driver_from_db(self, db_obj: Any) -> bool:
+  def construct_driver_from_db(self, db_obj: ConvolutionConfig) -> bool:
```

Comment:
leave it to Any, cause it can be both Conv and BatchNorn

Reply:
rolled back to Any